votingpool/pool.go

```diff
@@ -162,25 +202,32 @@ func LoadAndEmpowerSeries(namespace walletdb.Namespace, m *waddrmgr.Manager,
 
 // GetSeries returns the series with the given ID, or nil if it doesn't
 // exist.
-func (vp *Pool) GetSeries(seriesID uint32) *SeriesData {
-	series, exists := vp.seriesLookup[seriesID]
+func (p *Pool) GetSeries(seriesID uint32) *SeriesData {
```

Comment:
As pointed out in a previous comment, this should be `Series` instead of `GetSeries` per the Effective Go guidelines.